app/src/main/res/layout/main_fragment.xml

```diff
@@ -184,6 +184,46 @@
                 app:layout_constraintStart_toStartOf="@+id/start_guideline"
                 app:layout_constraintTop_toBottomOf="@+id/clear_session_button" />
 
+            <androidx.constraintlayout.widget.ConstraintLayout
+                android:id="@+id/visitor_code_switch_container"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                app:layout_constraintTop_toBottomOf="@id/visitor_code_button"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintEnd_toEndOf="parent" >
```

Comment:
NIT: would try to move this container a bit closer to the Visitor Code button. Maybe using negative margin or something else